lbrynet/daemon/Daemon.py

```diff
@@ -523,7 +523,8 @@ def get_est_cost_from_uri(self, uri):
         defer.returnValue(result)
 
     def get_est_cost(self, uri, size=None):
-        """Get a cost estimate for a lbry stream, if size is not provided the
+        """
+		Get a cost estimate for a lbry stream, if size is not provided the
         sd blob will be downloaded to determine the stream size
 
         """
```

Comment:
`Get a cost...` should be dedented to match how all of the other doc strings appear in `Daemon.py`. Also, please remove the blank line at the end.